### ISSUE TITLE
History Graph Card refresh_interval: has no effect.

### ISSUE THREAD

**Issue by del13r** [home-assistant/frontend#12999]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Since upgrading to 2022.6 and higher, I find that the history-graph card no longer scrolls/updates anymore.
I have to currently press refresh myself to see the updated data.
This is not ideal for a dashboard and defeats the purpose of having a refresh_interval specified.

### Describe the behavior you expected

I have a sensor that gets updated data every 60 seconds, however no matter how long I wait, the history-graph card does not automatically update.
I have to currently press refresh myself to see the updated data.
This was previously not a problem until upgrading to 2022.6 and higher.
<img width="187" alt="Screenshot 2022-06-23 160521" src="https://user-images.githubusercontent.com/86141523/175226948-47bbe506-ef28-4aac-9aa9-c77f8ad7c163.png">

### Steps to reproduce the issue

1. cleared browser cache
2. tried multiple browsers - chrome, edge.
3. Tried multiple computers and mobile phones.
4. Tried creating a new dashboard and setup a new history-graph card.
...

### What version of Home Assistant Core has the issue?

2022.6.7

### What was the last working version of Home Assistant Core?

2022.5

### In which browser are you experiencing the issue with?

Google Chrome Version 103.0.5060.53 (Official Build) (64-bit)

### Which operating system are you using to run this browser?

Windows 11 Pro x64 21H2 22000.739 

### Problem-relevant frontend configuration

```yaml
views:
  - title: Home
    cards:
      - type: history-graph
        entities:
          - entity: sensor.solar_power_corrected
          - entity: sensor.envoy_SERIALNUMBER_current_power_consumption
        refresh_interval: 60
        hours_to_show: 1
```

**Comment by ildar170975** [comment]
Probably this: https://github.com/home-assistant/frontend/issues/12859

**Comment by bdraco** [comment]
Yes this is a duplicate of https://github.com/home-assistant/frontend/issues/12859 which is fixed in 2022.7.x (not yet released)
